unikernel.ml

```diff
@@ -1383,13 +1303,8 @@ struct
                                         ~header_list:[ ("X-MOLLY-CSRF", csrf) ]
                                         `OK
                                   | Error err ->
-                                      reply reqd ~content_type:"text/html"
-                                        (Guest_layout.guest_layout
-                                           ~page_title:
-                                             "CSRF Token Error | Mollymawk"
-                                           ~content:
-                                             (Error_page.error_layout err)
-                                           ~icon:"/images/robur.png" ())
+                                      Middleware.http_response ~api_meth:false
+                                        reqd ~title:err.title ~data:err.data
```

Comment:
previously, the title was `"CSRF Token Error | Mollymawk"`, now it is `err.title`. Intentional?